packages/node_modules/@webex/internal-plugin-ediscovery/test/integration/spec/ediscovery.js

```diff
@@ -206,6 +207,48 @@ describe('internal-plugin-ediscovery', () => {
             assert.equal(error.statusCode, 0);
           });
       });
+
+      it('getContentContainer by url succeeds with valid url', async () => {
+        await complianceUser.webex.internal.ediscovery.getContentContainer(validUrl)
+          .then(() => {
+            assert.fail('Expected error response due to invalid report id');
+          })
+          .catch((error) => {
+            assert.equal(error.statusCode, 404);
+            assert.include(error.message, `Report Record with ID[${reportId}] could not be found`);
```

Comment:
Assume the service does return `Report Record` but I would have expected `Content Container`.

Reply:
The error is thrown because the report record does not exist